TRANSLATION.md

```diff
@@ -1,76 +1,25 @@
 # Node.js Website Translation Policy
 
-Node.js is a global platform and so this site has many translations. The translation of the site into
```

Comment:
I think the first part of "Node.js is a global platform and so this site has many translations. The translation of the site into languages other than English is handled by" can be kept. And just replace the part of translation groups to Crowdin